can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -0,0 +1,280 @@
+        dat = re.search("\d+;({.*})\d+;({.*})", r.text, re.MULTILINE)
+
+        # load info head and data group separately
+        info = json.loads(dat.group(1))
+        fdat = json.loads(dat.group(2))
+
+        # Get the county filter url params
+        cntFltr = getMapFilter(info)
+
+        # Get the state data:
+        outDf = outDf.append(extractData(fdat, self.state_fips), ignore_index=True)
+
+        if cntFltr and self.location_type == "county":
+            for county in self.cntys:
+                cntyReqParam = self.reqParams
+                for li in cntFltr:
+                    cntyReqParam[li] = county[0]
+                # You should be able to access an unlisted county-level tableau page at [r.url]
+                r = requests.get(url, params=cntyReqParam)
+                # Parse the output, return a json so we can build a bootstrap call
+                suppe = BeautifulSoup(r.text, "html.parser")
+                tdata = json.loads(
+                    suppe.find("textarea", {"id": "tsConfigContainer"}).text
+                )
+                # Call the bootstrapper: grab the state data, map selection update function
+                dataUrl = f'https://tableau.azdhs.gov{tdata["vizql_root"]}/bootstrapSession/sessions/{tdata["sessionid"]}'
+                r = requests.post(
+                    dataUrl,
+                    data={
+                        "sheet_id": tdata["sheetId"],
+                        "showParams": tdata["showParams"],
+                    },
+                )
+                # Regex the non-json output
+                dat = re.search("\d+;({.*})\d+;({.*})", r.text, re.MULTILINE)
+                # load info head and data group separately
+                info = json.loads(dat.group(1))
+                fdat = json.loads(dat.group(2))
+                # Get county data
+                cy = extractData(fdat, county[1])
+                outDf = outDf.append(cy, ignore_index=True)
+
+        outDf["CumPosTests"] = outDf["PercentPositive"] * outDf["Number of tests"]
+        outDf["CumDiagPosTests"] = (
+            outDf["Percent Positive Diagnostic tests"]
+            * outDf["Number of Diagnostic tests"]
+        )
+
+        # NOTE: There is currently a bug in the AZDHS dashboard summary page. They do NOT show correct values for antibody positivity rate
+        outDf["CumSeroPosTests"] = outDf["CumPosTests"] - outDf["CumDiagPosTests"]
+        if self.location_type == "county":
+            outDf = outDf[outDf["location"] != 4]
+        return outDf
+
+    def normalize(self, data):
+        df = data
+        crename = {
+            "New Cases": CMU(category="cases", measurement="new", unit="people"),
+            "New Deaths": CMU(category="deaths", measurement="new", unit="people"),
+            "New Tested": CMU(
+                category="unspecified_tests_total", measurement="new", unit="specimens"
+            ),
+            "New Tested PCR": CMU(
```

Comment:
`"New Tested PCR"` seems like it should be just `pcr_tests_total`?

Reply:
I thought so too, but according to the AZ dashboard, this column includes both PCR and antigen testing [(see the last '***' annotation](https://www.azdhs.gov/preparedness/epidemiology-disease-control/infectious-disease-epidemiology/covid-19/dashboards/index.php)).   The column name is determined from the Tableau-internal name.  I agree it's not super intuitive